modules/ipcCommunicator.js

```diff
@@ -4,6 +4,7 @@ Window communication
 @module ipcCommunicator
 */
 
+const _ = require('lodash');
```

Comment:
There are differences in the API, are you sure it doesn't break anything?
I mostly use `_.each` and `_.isEmpty`, which i saw exist as well.

My personal opinion is, yes it does have performance improvements, but for the use cases were we use it, it makes 0 difference..